spring-cloud-kubernetes-fabric8-autoconfig/src/main/java/org/springframework/cloud/kubernetes/fabric8/Fabric8Utils.java

```diff
@@ -39,14 +40,14 @@ private Fabric8Utils() {
 	private static final LogAccessor LOG = new LogAccessor(LogFactory.getLog(Fabric8Utils.class));
 
 	/**
-	 * this method does the namespace resolution for both config map and secrets
```

Comment:
I forgot to update this documentation a while ago, as it is re-used in other places as well, and will be used in discovery also at some point in time, its on my TODO list